x-pack/plugins/enterprise_search/public/applications/applications/components/engine/engine_connect/engine_api_integration.tsx

```diff
@@ -248,8 +248,7 @@ export const EngineApiIntegrationStage: React.FC = () => {
                 defaultMessage="To get the most out of the client, use the javascript client's example template and follow our {searchapplicationClientDocLink} on building a search experience."
                 values={{
                   searchapplicationClientDocLink: (
-                    // replace # with search application client doc link
-                    <EuiLink href="#">
+                    <EuiLink href={`${docLinks.searchApplications}`}>
```

Comment:
I think this should ideally take to getting started with browser client doc. For example: 
href={`${docLinks.searchApplications}` + '#replace-with-new-search-applications-browser-client-sub-header'}

Didn't add them as, I am not sure if above approach is acceptable. 

Also I think adding browser client docs is yet to be added. 
cc: @leemthompo

Reply:
Added in ff6e0cd8924e7d03268f91bebebd274cc4cf531c
Does it make sense to add an additional variable for `Search application getting started section`?

Reply:
yes this is a good idea, inline with what others have [done](https://github.com/elastic/kibana/blob/ff6e0cd8924e7d03268f91bebebd274cc4cf531c/packages/kbn-doc-links/src/get_doc_links.ts#L71)